index.html

```diff
@@ -0,0 +1,42 @@
+           <div id="capture">
+               <caption id="img-caption">Neque porro quisquam est qui dolorem ipsum quia dolor sit amet, consectetur, adipisci velit</caption>
+           </div>
+        </div>
+        </div>
```

Comment:
I think you should avoid overusing div elements. You can use semantic tags as they help organise your code and make it more SEO optimized. you can read more on semantic files here [http://diveintohtml5.it/semantics.html](url)

Reply:
- That's great you have included an explanation, a "why" for your suggestion. :+1:

Reply:
- This is a professional comment. The provision of a reference link makes it helpful to the author to read good resource about the subject matter. Good job:+1: